frontend/src/canvas/Canvas.js

```diff
@@ -544,6 +545,20 @@ const Canvas = props => {
     };
   }, [props.selectedColorId, pixelSelect, props.nftSelectionMode, nftSelectionStarted, nftSelectionPositionX, nftSelectionPositionY, nftSelectionWidth, nftSelectionHeight, height, width, props.canvasRef, nftSelectionEndX, nftSelectionEndY, nftSelectionStartX, nftSelectionStartY, props]);
 
+  // TODO: getLastPlacedPixelTime
+  useEffect(() => {
```

Comment:
Please update inside pixelsselector's placedTime instead of here with a new variable .

https://github.com/keep-starknet-strange/art-peace/blob/2dc9e5bd1240a4246317dc214d3c24e745b272d8/frontend/src/canvas/PixelSelector.js#L8